locales/en/app.ftl

```diff
@@ -823,3 +823,12 @@ additional-information-including = Additional information, including:
 
 # Title
 email-addresses-title = Email Addresses
+
+
+# This is a standardized breach overview blurb that appears on all breach detail pages.
+# $breachTitle is the name of the breached company or website.
+# $breachDate and $addedDate are calendar dates.
+breach-overview = On { $breachDate }, { $breachTitle } suffered a breach. Once the breach was discovered and verified it was added to our database on { $addedDate }.
```

Comment:
This loses the `<b>` tags around the title and dates, but I know adding HTML markup in the strings isn't always the best. Unless you're planning on injecting HTML markup around the title and localized dates, etc.

Reply:
It needed to go anyway.